<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QWidget" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16000</width>
    <height>9000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sea battle</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QMenuBar" name="menubar">
     <widget class="QMenu" name="menu">
      <property name="title">
       <string>Menu</string>
      </property>
      <addaction name="actionPreparePlayer_1"/>
      <addaction name="actionPreparePlayer_2"/>
      <addaction name="actionPlayer_1"/>
      <addaction name="actionPlayer_2"/>
     </widget>
     <addaction name="menu"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QWidget" name="wigdet_player_1" native="true">
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,9,1">
         <item>
          <widget class="QLabel" name="label_player_1">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Player 1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="GameWidget" name="game_area_1" native="true"/>
         </item>
         <item>
          <widget class="ShipListWidget" name="ships_list_1" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="wigdet_player_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,9,1">
         <item>
          <widget class="QLabel" name="label_player_2">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Player 2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="GameWidget" name="game_area_2" native="true"/>
         </item>
         <item>
          <widget class="ShipListWidget" name="ships_list_2" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionPreparePlayer_1">
   <property name="text">
    <string>Prepare player 1</string>
   </property>
  </action>
  <action name="actionPreparePlayer_2">
   <property name="text">
    <string>Prepare player 2</string>
   </property>
  </action>
  <action name="actionPlayer_1">
   <property name="text">
    <string>Player 1</string>
   </property>
  </action>
  <action name="actionPlayer_2">
   <property name="text">
    <string>Player 2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameWidget</class>
   <extends>QWidget</extends>
   <header>view.GameWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ShipListWidget</class>
   <extends>QWidget</extends>
   <header>view.ShipListWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
